ivy/functional/frontends/torch/utilities.py

```diff
@@ -0,0 +1,31 @@
+# global
+import torch
+
+# local
+import ivy
+
+
+def can_cast(from_, to):
+    res = 0
+    native_dtype_dict = {

```

Comment:
There is an ongoing task for type promotion on the PyTorch frontend I think it is better that you can wait for it to be done then you can go on with the task.

Reply:
Can I work on other issues if possible

Reply:
Yeah feel free to work on other task we will complete this one once type promotion is supported on the frontend side